<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sortingdialog</class>
 <widget class="QDialog" name="sortingdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting Tests</string>
  </property>
  <widget class="QPushButton" name="OpenFilePushButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort Array:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SortArrayLinedit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>741</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>Sort Algorithms:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QuickSortCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Quick Sort</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="MergeSortCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Merge Sort</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="BubbleSortCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Bubble Sort</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="RunSortTestsPushbutton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Run Sort Tests</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseSortTestsPushbutton">
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
